display/ssd1306.h

```diff
@@ -474,6 +474,10 @@ class StandardDisplayController : public DisplayControllerBase<Width, col_t>, Sa
   }
 
   void SB_On(EffectLocation location) override {
+    // Delay on.bmp until boot,font, or name message has been displayed for its full duration
```

Comment:
This doesn't actually "delay" anything, it just skips it.

Reply:
How about "avoid preempting images for boot, font, or name message, primarily when used with a blaster prop"

Reply:
I guess the question is really about how we want it to work?

Reply:
Yes. Would it be safe to assume that when switching presets the expectation would be to display the preset name or image, just like it does with a saber? And if a boot image exists it's likely intentional for the purpose of viewing when the blaster boots up. It's completely optional as always by just not having those images available if you don't want to see them. I think this fix is just aligning the behavior with what was intended initially and just never fleshed out? So how it's supposed to work seems rather intuitive to me. 🤷

Reply:
I was thinking that this might skip some step somewhere, but now that I look at it, I think it does not, so my concern isn't really valid.